marsha/settings.py

```diff
@@ -0,0 +1,102 @@
+"""Django settings for marsha project.
+
+Uses django-configurations to manage environments inheritance and the loading of some
+config from the environment
+
+"""
+
+import os
+
+from django.utils.translation import gettext_lazy as _
+
+from configurations import Configuration, values
+
+
+class Dev(Configuration):
+    """Django settings in Dev mode."""
+
```

Comment:
I usually define a `Base` configuration to inherit from. See Richie's example [1].

[1] https://github.com/openfun/richie/blob/master/richie/settings.py#L16